main.c

```diff
@@ -7,13 +7,15 @@
 #include "lib/network.c"
 #include "lib/pump.c"
 #include "lib/sensor.c"
+#include "lib/util.c"
 
 #define MAIN_QUEUE_SIZE     (8)
 static msg_t _main_msg_queue[MAIN_QUEUE_SIZE];
 
 static const shell_command_t shell_commands[] = {
     { "light", "read light data", read_light },
     { "humidity", "read humidity data", read_humidity },
+    { "udp_dumpd", "start debug udp server that just prints data it receives", dump_server },
```

Comment:
somehow, other RIOT packages can include their shell commands without adding a line, e.g. txtsnd. We could have a look at that.

Reply:
after milestone 1 :)